.circleci/config.yml

```
@@ -142,7 +142,7 @@ workflows:
       - unit_tests:
           matrix:
             parameters:
-              react-version: ["^17.0.0", "beta"]
+              react-version: ["^17.0.0", "^18"]
```

this semver range will not include prereleases; you may want `^18.0.0-rc`
```suggestion
              react-version: ["^17.0.0", "^18.0.0-rc"]
```

---

On line 82 it checks for this special case to do no installation (if [ "<< parameters.react-version >>" != "^18" ]; then). So this is just a descriptive name

---

gotcha, makes sense.